### PR TITLE
asdf 0.6.3

### PR DESCRIPTION
**Pull request by wulfmann**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Relevant File:
https://github.com/asdf-vm/asdf

### CODE REVIEW

File: Formula/asdf.rb [hunk 1]
```diff
@@ -29,10 +29,16 @@ def install
 
   def caveats; <<~EOS
     Add the following line to your bash profile (e.g. ~/.bashrc, ~/.profile, or ~/.bash_profile)
-         source #{opt_prefix}/asdf.sh
+    If you are using a framework, such as oh-my-zsh, place them at the end of your .zshrc
```
Comment (javian) [inline Formula/asdf.rb]: Shouldn't these be handled by the regular completion instructions ? See https://docs.brew.sh/Shell-Completion

**Comment by igas** [comment]
Thank you for your contribution @wulfmann. Brew using `caveats` for brew related info and not upstream documentation. As I can see this is basically a duplication of the readme. If @javian and @fxcoudert confirm this it's better to send PR with removing `caveats` altogether.

https://github.com/Homebrew/brew/blob/bada451a813031a0c7e5f063bea4358ecff5b2c6/Library/Homebrew/formula.rb#L1021-L1023

**Comment by wulfmann** [comment]
Sorry all, I guess I didn’t fully understand the purpose of the caveat section before. Having those instructions helped me during install so I figured I’d try and update them!

@javian are you saying you’d like me to add a shell completion section for those setup instructions? Or that it’s already in code somewhere?

Thanks for the quick response everyone

**Comment by igas** [comment]
Line 20 and 21 is where it should happen, if it does not we need to investigate. You can take a look at other formulas for inspiration. Do you @wulfmann want to do it in new PR with removing `caveats` and update bash/zsh/fish related installation steps in `install` block?

**Comment by wulfmann** [comment]
@igas sure I’ll update it and add a new or tomorrow. Thanks for the advice!

**Comment by wulfmann** [comment]
@igas Here is the new PR https://github.com/Homebrew/homebrew-core/pull/35767

**Pull request closed without merging**